tests/src/tools/ReadyToRun.SuperIlc/SuperIlcHelpers.cs

```diff
@@ -3,9 +3,126 @@
 using System.IO;
 using System.Linq;
 using System.Text;
+using System.Xml.Linq;
+using System.Xml.XPath;
+
+using Microsoft.Build.Construction;
+using Microsoft.Build.Evaluation;
 
 namespace ReadyToRun.SuperIlc
 {
+    /// <summary>
+    /// This class represents a single test exclusion read from the issues.targets file.
+    /// </summary>
+    public class TestExclusion
```

Comment:
Nit: I think these classes are worth having their own source file

Reply:
Good point, thanks, addressed in 2nd commit.